Library/ViewModels/PledgeViewModel.swift

```diff
@@ -860,75 +860,65 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
         )
       }
 
-    // Emmits value when Pledge Button is clicked.
+    // Signals events needed for when Pledge Button or Apple Pay Button is clicked.
```

Comment:
Maybe just change this to `Tracking` like we do in all the other files

Reply:
There's a `Tracking` at the top, I can just remove this comment entirely.